etc/profile-a-l/cointop.profile

```diff
@@ -0,0 +1,50 @@
+# Firejail profile for cointop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include cointop.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/cointop
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-X11.inc
+include disable-xdg.inc
+
```

Comment:
Can we use `include whitelist*common.inc`?

Reply:
I mean; we probably can?  But if the program runs fine without it what's the point in whitelisting all that common stuff?

Reply:
if it works, add them